### DISCUSSION TITLE
Go E Charger automatische Phasenumschaltung

### DISCUSSION THREAD

**Discussion by Trunks1982** [camueller/SmartApplianceEnabler#234] [Allgemein]
Hi Axel,
ich habe für den Go E die Http Api Adresse für die Phasenumschaltung könntest du das irgendwie einbauen?
Wäre für das testen natürlich bereit.

http://192.168.0.208/api/set?psm=2 3Phasig
http://192.168.0.208/api/set?psm=1 1Phasig

**Comment by camueller** [comment]
Bisher wird ja vor dem Einsschalten der Wallbox die Ladestromstärke gesetzt. Man könnte zusätzlich die Phasenanzahl setzen, welche ja schon für das Fahrzeug und/oder auch die Wallbox konfiguriert werden kann (bisher wird die Phasenanzahl nur für die Berechnung der Ladestromstärke verwendet).

Sobald es etwas zu Testen gibt, melde ich mich ...

**Reply by Trunks1982** [reply]
Super Danke👍

**Comment by Trunks1982** [comment]
@camueller weisst du schon wann du dafür Zeit hast?

**Reply by camueller** [reply]
Evtl. mache ich das nach den Multi-Switches für die nächste Version. Mal sehen ...

**Reply by Trunks1982** [reply]
Ok.Baust du ne Verzögerung mit ein für wolken?

**Reply by camueller** [reply]
Was hat denn das Setzen der Pahsenanzahl mit Wolken zu tun? Einbau wie oben beschrieben.

**Reply by Trunks1982** [reply]
Dachte eher das es wie beim evcharger funktioniert zb über 3800w bei den 11Kw Chargern das dann auf 3phasig umgeschaltet wird.So hätte man die komplette Range von 1380w bis 11000w.

**Reply by camueller** [reply]
Daran hatte ich nicht gedacht, aber so könnte man es auch machen. Guter Hinweis!

**Reply by Trunks1982** [reply]
Nicht schlimm da haben wir uns etwas missverstanden🤣
